packages/build/config/.mocharc.json

```diff
@@ -1,7 +1,9 @@
 {
-  "require": "source-map-support/register",
+  "require": ["source-map-support/register"],
   "recursive": true,
   "exit": true,
-  "reporter": "dot"
+  "reporter": "dot",
+  "timeout": 5000,
+  "parallel": true
```

Comment:
I am against forcing all LB users to adopt parallel testing mode. As you know from our own experience, adapting a test suite to support parallel execution is a non-trivial amount of work. The situation is even more complicated when stateful 3rd-party services are involved, typically when the test suite works with data stored in a real database. (We are not affected much, because most of our tests are using in-memory datasources.)

In smaller projects, the cost of enabling parallel testing is often not worth the benefits.

I am concerned that by making parallel testing the default in new projects, we will make the framework more difficult to adopt for less-experienced developers.